pkg/controllers/tagging/tagging_controller.go

```diff
@@ -105,9 +111,29 @@ func NewTaggingController(
 	// Use shared informer to listen to add/update/delete of nodes. Note that any nodes
 	// that exist before tagging controller starts will show up in the update method
 	tc.nodeInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
-		AddFunc:    func(obj interface{}) { tc.enqueueNode(obj, tc.tagNodesResources) },
-		UpdateFunc: func(oldObj, newObj interface{}) { tc.enqueueNode(newObj, tc.tagNodesResources) },
-		DeleteFunc: func(obj interface{}) { tc.enqueueNode(obj, tc.untagNodeResources) },
+		AddFunc: func(obj interface{}) {
+			node := obj.(*v1.Node)
+			tc.enqueueNode(node, tc.tagNodesResources)
+		},
+		UpdateFunc: func(oldObj, newObj interface{}) {
+			node := newObj.(*v1.Node)
+			// We have a check for this when tagging instances too. The reason this check exists here is to make sure that
```

Comment:
> We have a check for this

Use more descriptive wording than `this`.  It seems like its explained partially in the next sentence so maybe just reword so the explanation doesn't come after the first sentence.  E.g.:

> Check if tagging is required by converting desired tags to labels and checking if node has already been labelled.  The reason this check exists...